interfaces/vrep/DQ_VrepInterface.m

```diff
@@ -227,24 +248,44 @@ function stop_simulation(obj)
             end
         end
         
-        %% Get Handle
+        
         function handle = get_handle(obj,name)
-            %% Get the V-REP handle for a given object
+            % This method gets the server handle for a given object. 
+            % object names.
+            % Usage:
+            %     get_handles(name);  
+            %          names: The object name.
+            % Example: 
+            %     handle = get_handle('ReferenceFrame');
+
             [~,handle] = obj.vrep.simxGetObjectHandle(...
                 obj.clientID,...
                 name,...
                 obj.vrep.simx_opmode_blocking);
         end
         
-        %% Get Object Translation
-        function t = get_object_translation(obj,handle,relative_to_handle,opmode)
-            %% Get the translation of an object in V-REP
-            %%  >> t = vi.get_object_translation('DefaultCamera');
+        
+        function t = get_object_translation(obj,objectname,reference_frame,opmode)
+            % This method gets the translation of an object in the server
+            %
+            % Usage:
+            %     t = get_object_translation(objectname,reference_frame,opmode);  
+            %          objectname:  The object name
+            %          (optional) reference_frame:  Indicates the name of 
+            %                       the relative reference frame in which you 
+            %                       want the translation. If not specified, 
+            %                       the absolute frame is used.
+            %          (optional) opmode:The operation mode.             
```

Comment:
@juanjqo Just adding that it’s probably better to make it explicit in the documentation, considering the additions mentioned by @bvadorno, that leaving this empty is the recommended way to use these methods. Because if left empty the opmode will be automatically managed by our library and using anything else is considered “advanced”.

Reply:
Hi @bvadorno and @mmmarinho 

I modified the documentation taking into account your comments. Now, we have

 ```matlab
 function set_joint_target_velocities(obj,jointnames,joint_target_velocities,opmode)
            % This method sets the joint velocities of a robot in the CoppeliaSim scene.
            % It is required a dynamics enabled scene, and joints in dynamic mode 
            % with velocity control mode. Check this link for more
            % information about joint modes:
            % https://www.coppeliarobotics.com/helpFiles/en/jointModes.htm
            %
            % Usage:
            %      Recommended:
            %      set_joint_target_velocities(jointnames, joint_target_velocities);
            %
            %      Advanced:
            %      set_joint_target_velocities(jointnames, joint_target_velocities, opmode);   
            %
            %          jointnames: The joint names.
            %          joint_target_velocities: The joint target velocities.
            %          (optional) opmode: The operation mode. If not specified, 
            %                       the opmode will be set automatically. 
            %                          
            %                        You can use the following modes:
            %                           OP_BLOCKING 
            %                           OP_STREAMING 
            %                           OP_ONESHOT 
            %                           OP_BUFFER;
            %
            %                       Check this link for more details:
            %                       https://www.coppeliarobotics.com/helpFiles/en/remoteApiModusOperandi.htm
            %
            % Example:
            %      jointnames={'LBR4p_joint1','LBR4p_joint2','LBR4p_joint3','LBR4p_joint4',...
            %                  'LBR4p_joint5','LBR4p_joint6','LBR4p_joint7'};
            %      u = [0.1 0.1 0.1 0.1 0.1 0.1 0.1];
            %
            %      % Recommended:
            %      set_joint_target_velocities(jointnames, u);
            %
            %      % Advanced usage:
            %      set_joint_target_velocities(jointnames, u, OP_ONESHOT);
            .
            .
            .
end
```